content/bitcoin-core.md

```diff
@@ -38,6 +38,14 @@ If ASmap is enabled, the `getpeerinfo` RPC command's response will include `mapp
 
 Similarly, the `getrawaddrman` RPC command's response will include `source_mapped_as` field, indicating which AS this peer's source IP was mapped to, if any (as of Bitcoin Core [v28.0](https://github.com/bitcoin/bitcoin/blob/1147e72953d1f262111a4b1d5a438a8394511bc7/src/rpc/net.cpp#L1160)).
 
+#### Measuring coverage
```

Comment:
I think that's the wrong headline here. Measuring coverage is what kartograf cov does IMO: https://github.com/asmap/kartograf?tab=readme-ov-file#check-coverage-of-a-mapping-file. This I would call "Inspecting differences in maps" or something like that.

Reply:
went with the slightly more verbose "Inspecting assignment changes between two ASmaps"